client/pages/waitList.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import WaitList from '../components/waitList';
+
+export default function Home(props) {
+  return (
+    <div>
+      <WaitList />
+    </div>
+  );
+}
```

Comment:
I don't think this file is necessary. If it is, please move your waitlist component into pages and use it directly instead of this Home component